rch/src/main/java/org/motechproject/nms/rch/service/impl/RchWebServiceFacadeImpl.java

```diff
@@ -519,7 +531,9 @@ private void districtFileProcess(String fileName, String result, Long stateId, S
             rchImportAuditDataService.create(new RchImportAudit(startDate, endDate, RchUserType.DISTRICT, stateCode, stateName, 0, 0, error));
             rchImportFailRecordDataService.create(new RchImportFailRecord(endDate, RchUserType.DISTRICT, stateId));
         } catch (NullPointerException e) {
```

Comment:
Shall we change all NullPointerExce to just Exception and "No files saved a" to "No files saved TYPE + e.getMessage () "

Reply:
ok

Reply:
Lets go ahead with exception